src/typings.ts

```diff
@@ -356,14 +360,33 @@ export interface DecimalsObject {
 export interface UserObject {
   address: string
   pubKey: string
-  keystore: string
+  backup: string
```

Comment:
This breaks backwards compatibility, do we really need that?
If we break like this we should make it clear in the PR or Commit message.